pkg/policy/validate.go

```diff
@@ -106,6 +106,10 @@ func Validate(policy *kyverno.ClusterPolicy, client *dclient.Client, mock bool,
 		clusterResourcesMap := make(map[string]*struct{})
 		// Get all the cluster type kind supported by cluster
 
+		if len(policy.Spec.ValidationFailureActionOverrides) > 0 {
+			return fmt.Errorf("invalid policy: use of ValidationFailureActionOverrides in a Namespace Policy")
```

Comment:
nit: can we improve this error message , saying why its invalid ?

Reply:
Pretty much this, because the ValidationFailureActionOverrides is only defined for Cluster Policies.